config/channels/in-memory-channel/300-in-memory-channel.yaml

```diff
@@ -22,15 +22,38 @@ metadata:
     duck.knative.dev/addressable: "true"
 spec:
   group: messaging.knative.dev
-  preserveUnknownFields: false
-  validation:
-    openAPIV3Schema:
-      type: object
-      # this is a work around so we don't need to flush out the
-      # schema for each version at this time
-      #
-      # see issue: https://github.com/knative/serving/issues/912
-      x-kubernetes-preserve-unknown-fields: true
+  versions:
+  - &version
```

Comment:
:scream:

Reply:
well, we do that already with `everything` on the verbs ;-) 

but yeah :roll_eyes:

Reply:
Nah, it is fine :) Just some YAML magic that I didn't know before :)
Actually gets rid of some dupes, which is nice